utils/obsproc/NetCDFToIodaConverter.h

```diff
@@ -0,0 +1,61 @@
+#ifndef GDAS_UTILS_OBSPROC_NETCDFTOIODACONVERTER_H_  //NOLINT
```

Comment:
```suggestion
#pragma once
```

Reply:
Just pragma once and remove all the ifndef stuff

Reply:
Ha! yes ... dummy me. Are we sure it's doing the same as the ifndef stuff?

Reply:
Indeed, ask ChatGPT to confirm though :-)

Reply:
Just did, the only issue is portability. It's not part of the c++ standard but it's widely supported. Works for me!